contracts/deploy/sonic/024_increase_timelock_delay.js

```diff
@@ -0,0 +1,41 @@
+const addresses = require("../../utils/addresses.js");
+const { deployOnSonic } = require("../../utils/deploy-l2.js");
+
+module.exports = deployOnSonic(
+  {
+    deployName: "024_increase_timelock_delay",
+    forceSkip: false,
+  },
+  async ({ ethers }) => {
+    // 1. Deploy new OS implementation
+    const cARMProxy = await ethers.getContractAt(
+      ["function setOwner(address) external"],
+      "0x2F872623d1E1Af5835b08b0E49aAd2d81d649D30"
+    );
+    const cTimelock = await ethers.getContractAt(
+      "ITimelockController",
+      addresses.sonic.timelock
+    );
+
+    console.log("ARM Proxy:", cARMProxy.address);
+    console.log("Timelock:", cTimelock.address);
+
+    return {
+      name: "Upgrade OS token contract",
+      actions: [
+        // 1. Upgrade the OSonic proxy to the new implementation
+        //{
+        //  contract: cARMProxy,
+        //  signature: "setOwner(address)",
+        //  args: [addresses.sonic.timelock],
+        //},
```

Comment:
nit: Could remove the commented out lines, this has to go through the multisig and not Timelock anyway

Reply:
Indeed, I forget to remove those. Changed in this commit: [3ec33d0](https://github.com/OriginProtocol/origin-dollar/pull/2606/commits/3ec33d0aac372e3af9684a4da840e813316aab2c)